python/cugraph-pyg/cugraph_pyg/tests/test_cugraph_loader.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import cupy
+import cudf
+
+from cugraph.experimental.datasets import karate
+from cugraph.gnn import FeatureStore
+from cugraph_pyg.loader import CuGraphNeighborLoader
+from cugraph_pyg.data import CuGraphStore
+
+import pytest
+
+
+@pytest.fixture
+def karate_gnn():
```

Comment:
Maybe a FIXME, but can this be moved to a common place (`conftest.py`?) so that it doesn't need to be copied, or is it actually different than the other `karate_gnn()` fixture?  Same for the other fixtures which look familiar.

Reply:
done